common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -4,7 +4,7 @@
 <div id="formulaequationinput_${id}" class="inputtype formulaequationinput" ${doinline | n, decode.utf8}>
     <div class="${status.classname}" id="status_${id}">
       % if response_data['label']:
-          <label class="problem-group-label" for="input_${id}">${response_data['label']}</label>
+          <label class="problem-group-label" for="input_${id}">${HTML(response_data['label'])}</label>
       % endif
         % for description_id, description_text in response_data['descriptions'].items():
             <p class="question-description" id="${description_id}">${description_text}</p>
```

Comment:
Is description now HTML due to stringify, or am I not following this code right?